cdf-core/cdf/js-modules/components/ButtonComponent.js

```diff
@@ -45,6 +75,7 @@ define(['amd!../lib/underscore', '../lib/jquery', './ActionComponent'],
      */
     disable: function() {
       this.placeholder('button').attr('disabled', 'disabled');
+      this.placeholder('button').css({ opacity: 0.5 });
     },
```

Comment:
Probably best if we assign a css class to the button and have this style defined in cdf.css (not sure about the name). That way it can be overriden by implementors